llmfoundry/utils/__init__.py

```diff
@@ -13,23 +13,18 @@
 # yapf: disable
 from llmfoundry.utils.model_download_utils import (
     download_from_hf_hub, download_from_http_fileserver)
+from llmfoundry.utils.mosaicmllogger_utils import (find_mosaicml_logger,
+                                                   log_eval_analytics,
+                                                   log_train_analytics)
 
 # yapf: enable
 
 __all__ = [
-    'build_callback',
-    'build_logger',
-    'build_algorithm',
-    'build_optimizer',
-    'build_scheduler',
-    'build_icl_evaluators',
-    'build_tokenizer',
-    'calculate_batch_size_info',
-    'convert_and_save_ft_weights',
-    'download_from_http_fileserver',
-    'download_from_hf_hub',
-    'get_hf_tokenizer_from_composer_state_dict',
-    'update_batch_size_info',
-    'log_config',
-    'pop_config',
+    'build_callback', 'build_logger', 'build_algorithm', 'build_optimizer',
+    'build_scheduler', 'build_icl_evaluators', 'build_tokenizer',
+    'calculate_batch_size_info', 'convert_and_save_ft_weights',
+    'download_from_http_fileserver', 'download_from_hf_hub',
+    'get_hf_tokenizer_from_composer_state_dict', 'update_batch_size_info',
+    'log_config', 'pop_config', 'find_mosaicml_logger', 'log_eval_analytics',
+    'log_train_analytics'
```

Comment:
```suggestion
    'log_train_analytics',
```

add a trailing comma so the auto format is nice